src/full/Agda/Utils/Pretty/Aspect.hs

```diff
@@ -0,0 +1,174 @@
+module Agda.Utils.Pretty.Aspect where
```

Comment:
Since this module contains Agda language specific definitions, I'd be more happy to see it placed in the `Agda.Syntax` tree, and keep `Agda.Utils` for mostly general utilities enhancing what is missing in the Haskell ecosystem (or not packaged optimally for us).

Reply:
I put it here because I didn't want to change every reference to `Doc`, but moving `Agda.Utils.Pretty` didn't feel right, either. But I agree it feels... wrong. Does the following sound better:

```
Agda.Utils.Pretty → Agda.Syntax.Pretty
Agda.Utils.Pretty.Aspect → Agda.Syntax.Pretty.Aspect
Agda.Utils.Pretty.ANSI → Agda.Syntax.Pretty.ANSI
```

If so, I'm happy to make these moves.

Reply:
I would support placing them in `Agda.Syntax.Common`, like `Agda.Syntax.Common.{ANSI,Aspect,Pretty}`.